DeepSpeech.py

```diff
@@ -1853,7 +1855,10 @@ def export():
         tf.reset_default_graph()
         session = tf.Session(config=session_config)
 
-        inputs, outputs = create_inference_graph(batch_size=1, n_steps=FLAGS.n_steps, tflite=FLAGS.export_tflite)
+        export_batch_size = 1
+        window_size = 2*n_context+1
```

Comment:
No need to hard-code this, get it from the shape of the input tensor returned by `create_inference_graph` to avoid coupling as well as this code being out of sync with `create_inference_graph`.

```python
inputs, outputs = create_inference_graph(...)
input_names = [",".join(tensor.op.name for tensor in inputs.values())]
input_shapes = [":".join(",".join(map(str, tensor.shape)) for tensor in inputs.values())]
```

(And then remove 'input_lengths' from the returned dictionary when tflite is being used).